README.md

```diff
@@ -46,9 +46,9 @@ We’re excited to get early feedback and to continue working on more use cases!
 Bottlerocket is architected such that different cloud environments and container orchestrators can be supported in the future.
 A build of Bottlerocket that supports different features or integration characteristics is known as a 'variant'.
 The artifacts of a build will include the architecture and variant name.
-For example, an `x86_64` build of the `aws-k8s-1.17` variant will produce an image named `bottlerocket-aws-k8s-1.17-x86_64-<version>-<commit>.img`.
+For example, an `x86_64` build of the `aws-k8s-1.19` variant will produce an image named `bottlerocket-aws-k8s-1.19-x86_64-<version>-<commit>.img`.
 
-Our first supported variants, `aws-k8s-1.15`, `aws-k8s-1.16`, and `aws-k8s-1.17`, support EKS as described above.
+Our first supported variants, `aws-k8s-1.15`, `aws-k8s-1.16`, `aws-k8s-1.17`, `aws-k8s-1.18`, `aws-k8s-1.19` support EKS as described above.
```

Comment:
Nit: this kinda feels weird to read. It made more sense when Bottlerocket first launched. But now that we're releasing new variants maybe..
```suggestion
`aws-k8s-1.15`, `aws-k8s-1.16`, `aws-k8s-1.17`, `aws-k8s-1.18`, `aws-k8s-1.19` support EKS as described above.
```

Reply:
I agree but opted for a light touch. I'll look at your suggestion and see what we can do.

Reply:
Paging @tjkirch, please see my re-wording of these sentences in the README. Thanks!